### PR TITLE
Allow to update animation duration

### PR DESCRIPTION
**Pull request by krzysztofpelczarinfullmobile**
Allow to update animation duration.
It allows to set the initial value without animation and enable animation when user interacts with view.

### CODE REVIEW

File: lib/flutter_animation_progress_bar.dart [hunk 1]
```diff
@@ -77,6 +77,7 @@ class _FAProgressBarState extends State<FAProgressBar>
           .animate(_controller);
     });
     _controller.reset();
+    _controller.duration = widget.animatedDuration;
```
Comment (ltdangkhoa) [inline lib/flutter_animation_progress_bar.dart:80]: I think to duration has already set at line 53

**Review by ltdangkhoa — COMMENTED** [review]
Do you think it's a good idea to add this line of code?

File: lib/flutter_animation_progress_bar.dart (hunk 1 above)
Comment (krzysztofpelczarinfullmobile) [reply lib/flutter_animation_progress_bar.dart:80]: Yes. It is set when the progress bar is initialized.
Re-setting it here allows to change the duration later (if value is the same it does nothing).
In my use case I have to show progress when screen is opened without animation. Later user can interact with the screen and the value change is animated.

Comment (ltdangkhoa) [reply lib/flutter_animation_progress_bar.dart:80]: Thanks for your clarifying, I'll merge to the main branch and publish new version soon

Comment (krzysztofpelczarinfullmobile) [reply lib/flutter_animation_progress_bar.dart:80]: It's awesome. Thank you 🎉

**Pull request merged**